xmrstak/cli/cli-miner.cpp

```diff
@@ -90,6 +89,170 @@ void help()
 	cout<<"Brought to by fireice_uk and psychocrypt under GPLv3."<<endl;
 }
 
+bool read_yes_no(const char* str)
```

Comment:
please use `const std::string str` as parameter

Reply:
Those are pointers to read-only memory loaded with the binary. There is no point to wrapping them in objects as we will never write to them (str here is just a const string printed to the screen as user prompt)